person.rb

```diff
@@ -0,0 +1,24 @@
+class Person
+  attr_reader :id, :name, :age
```

Comment:
- [ ]   # :name and :age instance variables are the only ones that should be able to be modified that's why they are supposed to be defined as attr_accessors (here both Getters and Setters are defined), :id should only be able to be retrieved so only the Getter should be defined